server/network.js

```diff
@@ -120,14 +134,37 @@ network.query.combinedRegulation = function(query, networkPath) {
 };
 
 /**
- * @param {string} networkAddr
+ * @param {!network.query.AddGene} query
+ * @param {string} networkPath
+ * @return {!Array<!network.Edge>}
+ */
+network.query.addGene = function(query, networkPath) {
+  var fileName = query.fileName;
+  var gene = query.gene;
+  var file = networkPath + fileName;
+  var nodes = query.nodes;
+  return network.addGene_(file, gene, nodes);
+};
+
+/**
+ * @param {string} networkPath
  * @return {!Array<{
  *   networkName: string,
  *   description: string
  * }>}
  */
-network.query.list = function(networkAddr) {
-  return network.listNetwork_(networkAddr);
+network.query.list = function(networkPath) {
+  return network.listNetwork_(networkPath);
+};
+
+/**
+ * @param {!network.query.NetworkInfo} query
+ * @param {string} networkPath
+ * @return {!Array<!network.NetworkNode>}
```

Comment:
I made a mistake that didn't include server compilation. This is fixed in #76. After merging #76 please sync again and rerun the compilation.